import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/cstore/multipledest/sender/CStoreSenderTest.java

```diff
@@ -123,24 +124,24 @@ public JSONArray qidoRs(String path) throws DicomWebException {
 
     rspAssert.assertResult();
 
-    inputStream = Files.newInputStream(Paths.get(pathToDcmFile));
+    inputStream = Files.newInputStream(Paths.get(PATH_TO_DCM_FILE));
```

Comment:
I'm trying to understand why we do a cmove here?

Reply:
I needed any dicom test endpoint with stub. This one i found in tests that already implemented.

Reply:
Could we remove the code that calls cmove on the association and just call cstore in the sendFile test? Isn't the stub for cstore all we need?

Reply:
yes, fixed.